src/parseInputs.js

```diff
@@ -21,8 +22,18 @@ export async function parseInputs(cliInputs) {
 
   let useComponent = () => {
     component = true;
+    if (starlingMonkey) {
+      noStarlingMonkeyComponent();
+    }
     wasmEngine = join(__dirname, "../js-compute-runtime-component.wasm");
   };
+  let useStarlingMonkey = () => {
+    starlingMonkey = true;
+    if (component) {
+      noStarlingMonkeyComponent();
+    }
+    wasmEngine = wasmEngine = join(__dirname, "../starling.wasm");
```

Comment:
Could we check for the existence of the starling monkey wasm, and give a more specific error message if it's not present?

Reply:
Added.